tests/test_rst.py

```diff
@@ -624,6 +624,28 @@ def func(): pass
 """
 
 
+# https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists
+SPHINX_SIGNATURE = """
```

Comment:
Is there code that handles any code blocks before this? Otherwise something like 

```
.. code-block:: python
    def foo():
        """:param  str message_body: blah blah"""
```

could cause problems. I'll admit it's an edge case that would likely only come up in docstring parsing code.

Reply:
Oh, one other thing that comes to mind! If the programmer put these directives in a random order for some reason, it would be nice if all directives of a certain type were grouped together. Also definitely a bit of an edge case & not required, but there's plenty of less-than-finely crafted code out there.

Reply:
No, it will not cause problems. Code blocks are properly parsed out at an earlier stage. I added a test case in 1771df2f1472266de9b8fb04c87111d1baea11b5 to demonstrate this.

Reply:
I agree about the order thing. That would require a proper parser though so we cannot just use regexp here. It is definitely in scope, but I don't have more time this weekend. Would you mind opening an issue so we can track this as a future improvement?

Reply:
resolved with https://github.com/python-lsp/docstring-to-markdown/issues/14